### ISSUE TITLE
Tests fail with Net::DNS 1.03 (Net-DNS-Paranoid-0.08)

### ISSUE THREAD

**Issue by eserte** [tokuhirom/Net-DNS-Paranoid#2]
t/01-simple.t test fails, possibly only if Net::DNS 1.03 is installed (Net::DNS 1.02 and earlier is fine).

```
Output from './Build test':

t/00_compile.t .. ok
Not a GLOB reference at lib/Net/DNS/Paranoid.pm line 51.
    # Child (this domain is okay.  50.112.116.235.xip.io isn't blocked) exited without calling finalize()

#   Failed test 'this domain is okay.  50.112.116.235.xip.io isn't blocked'
#   at /opt/perl-5.22.1-RC1/lib/site_perl/5.22.1/Test/Builder.pm line 279.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 255 just after 10.
t/01_simple.t ... 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 1/10 subtests 
```

**Comment by wtoorop** [comment]
The soon to be released Net::DNS 1.04 will make it work again

**Comment by eserte** [comment]
Yes, it seems that there are no 100% failures with Net::DNS 1.04 and up anymore.
